cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -1045,7 +1045,8 @@ def table_of_content(self, key, value):
     ).strip()
     if text != "--":
         chapters = re.split(r"; | -- |--", text)
-        return chapters
+        chapters = [elem.strip(' ') for elem in chapters]
```

Comment:
for this you can use the decorator function, if I remember correctly it was `@out_strip`, automatically cleans the list of empty values